### PR TITLE
Update READMEs 

### PR DESCRIPTION
**Pull request by johnsonmh**

### CODE REVIEW

File: google_fonts/README.md [hunk 1]
```diff
@@ -44,23 +46,32 @@ or
...
-To override the fontWeight and/or fontStyle:
+To override the `fontSize`, `fontWeight`, or `fontStyle`:
  
 ```dart
 Text(
   'This is Google Fonts',
-  style: GoogleFonts.lato(
+  style: GoogleFonts.latoTextStyle(
     textStyle: Theme.of(context).textTheme.display1,
+    fontSize:48,
```
Comment (clocksmith) [inline google_fonts/README.md]: nit space

File: google_fonts/README.md [hunk 2]
```diff
@@ -44,23 +46,32 @@ or
...
 ),
 ```
+
+You can also use `GoogleFonts.latoTextTheme()` to make or modify an entire text theme to use the "Lato" font.
+
+```dart
+final textTheme = GoogleFonts.latoTextTheme(
+    textTheme: Theme.of(context).textTheme,
+);
+```
```
Comment (clocksmith) [inline google_fonts/README.md]: maybe an example of copyWith to have different families for body1 and body2?

File: google_fonts/README.md (hunk 1 above)
Comment (johnsonmh) [reply google_fonts/README.md]: Done.

File: google_fonts/README.md (hunk 2 above)
Comment (johnsonmh) [reply google_fonts/README.md]: Done.

**Pull request merged**